pandas/core/indexes/datetimelike.py

```diff
@@ -342,7 +342,8 @@ def _format_with_header(self, header, **kwargs):
     def __contains__(self, key):
         try:
             res = self.get_loc(key)
-            return is_scalar(res) or type(res) == slice or np.any(res)
+            return is_scalar(res) or isinstance(res, slice) or \
```

Comment:
Typically we use parenthesis to break long lines, not `\`.

Reply:
Changed that

Reply:
Also fixed the conflict